From 21a6b773e077d89937b59885243a59f8e9b079d4 Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Wed, 22 Mar 2017 11:33:38 +0800
Subject: [PATCH 1606/5242] MLK-14498-11 reset: gpio-reset: use the cansleep
 variant of the GPIO API

commit  c7cbfe3fcb3b0deec37507076716be0de8fb2fac from
https://source.codeaurora.org/external/imx/linux-imx.git

Use the cansleep variant of the GPIO API.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/reset/gpio-reset.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/reset/gpio-reset.c b/drivers/reset/gpio-reset.c
index 4cb9929..3049b22 100644
--- a/drivers/reset/gpio-reset.c
+++ b/drivers/reset/gpio-reset.c
@@ -32,7 +32,7 @@ static void gpio_reset_set(struct reset_controller_dev *rcdev, int asserted)
 	if (drvdata->active_low)
 		value = !value;
 
-	gpio_set_value(drvdata->gpio, value);
+	gpio_set_value_cansleep(drvdata->gpio, value);
 }
 
 static int gpio_reset(struct reset_controller_dev *rcdev, unsigned long id)
-- 
1.7.9.5

